timesketch/api/v1/resources/archive.py

```diff
@@ -470,21 +470,48 @@ def _export_view(self, view, sketch, zip_file):
         if not indices or '_all' in indices:
             indices = self.sketch_indices
 
+        # Ignoring the size limits in views to reduce the amount of queries
+        # needed to get all the data.
+        query_filter['terminate_after'] = 10000
+        query_filter['size'] = 10000
+
         query_dsl = view.query_dsl
         if query_dsl:
             query_dict = json.loads(query_dsl)
             if not query_dict:
                 query_dsl = None
 
-        # TODO (kiddi): Enable scrolling support.
         result = self.datastore.search(
             sketch_id=sketch.id,
             query_string=view.query_string,
             query_filter=query_filter,
             query_dsl=query_dsl,
+            enable_scroll=True,
             indices=indices)
 
-        fh = _query_results_to_filehandle(result, sketch)
+        scroll_id = result.get('_scroll_id', '')
+        if scroll_id:
+            data_frame = _query_results_to_dataframe(result, sketch)
+
+            total_count = result.get(
+                'hits', {}).get('total', {}).get('value', '')
```

Comment:
What if this fails? (and `total_count` is `''`)? The comparison in line 500 will raise an exception

Reply:
yes, changed a bit